### ISSUE TITLE
Update to Python 3.8

### ISSUE THREAD

**Issue by Agatha2322** [ibmdb/python-ibmdb#462]
Hi, 

Is there any option to update ibm-db to Python 3.8 soon? 
I saw that last update was in march 2019 - few months before reveal 3.8 

Update will be helpfull

**Comment by SabaKauser** [comment]
Hi, Yes. I will add support for python 3.8. May be in early March.

**Comment by soundstripe** [comment]
Wouldn't it would be better to leave this issue open as it is not resolved?

**Comment by SabaKauser** [comment]
I plan to have the release and hence closed it. If you want it to be in open state, I have no issues.

**Comment by soundstripe** [comment]
Its your package but I was searching for the same answer and nearly opened a new issue as I didn't see this one listed. No worries either way. Thanks for your work on this!

**Comment by rsivaibm** [comment]
Is this issue fixed in the latest release ?

**Comment by SabaKauser** [comment]
no. I need to make a new release for python 3.8 support.

**Comment by rsivaibm** [comment]
Does 3.0.1 release work with 3.7 python ?

**Comment by SabaKauser** [comment]
yes. it works.

**Comment by Veronit** [comment]
@SabaKauser You wouldn't happen to have an update on the timetable?

**Comment by SabaKauser** [comment]
@Veronit , I am occupied delivering other priorities for new release of other product. I will have time as soon as I am done with that. Next item in my list is new release of python driver with 3.8 support.

**Comment by Veronit** [comment]
Nice. Thanks for the info!

**Comment by byegates** [comment]
I am also waiting on 3.8 support on windows, looking forward to your update!

**Comment by SabaKauser** [comment]
The support is available. new version is 3.0.2.
